make/devkit/createWindowsDevkit.sh

```diff
@@ -60,12 +60,18 @@ UNAME_RELEASE=`uname -r`
 # Detect cygwin or WSL
 IS_CYGWIN=`echo $UNAME_SYSTEM | grep -i CYGWIN`
 IS_WSL=`echo $UNAME_RELEASE | grep Microsoft`
+IS_MSYS=`echo $UNAME_SYSTEM | grep MSYS_NT`
 if test "x$IS_CYGWIN" != "x"; then
     BUILD_ENV="cygwin"
+    CMD_EXE="cmd.exe /c"
+elif test "x$IS_MSYS" != "x"; then
+    BUILD_ENV="cygwin"
+    CMD_EXE="cmd.exe //c"
```

Comment:
The `//c` is a hacky solution. 

I think it would be better if you exported `MSYS2_ARG_CONV_EXCL="*"` -- you can do that without testing for windows environment since it does not really matter on anything but msys.

Then you can skip `CMD_EXE` and keep the single-slash variant for all environments.

Reply:
Thanks for your comments. I added `MSYS2_ARG_CONV_EXCL="*"` in script and tested with msys.